modules/dns_dnscmd/dns_dnscmd_main.rb

```diff
@@ -67,7 +73,7 @@ def execute cmd, msg=nil, error_only=false
         std_err.close unless std_in.nil?
       end
       report msg, response, error_only
-      response
+      response if return_result
```

Comment:
Good to know, Foreman expects `nil`. I think our test unit is getting better by the minute:  use `assert_nil`.